modules/kafka-cluster/main.tf

```diff
@@ -10,16 +11,27 @@ resource "confluent_kafka_cluster" "cluster" {
   region       = var.region
 
   dynamic "basic" {
-    for_each = local.build_cluster_type_basic
+    for_each = var.cluster_type == "basic" ? [1] : []
     content {}
   }
 
   dynamic "standard" {
-    for_each = local.build_cluster_type_standard
+    for_each = var.cluster_type == "standard" ? [1] : []
     content {}
   }
-
+  dynamic "dedicated" {
+    for_each = var.cluster_type == "dedicated" ? [1] : []
+    content {
+      cku = var.cku
+    }
+  }
```

Comment:
```suggestion
  }

```

Reply:
@MXfive I have fixed this issues by updating permission hence pre-commit ran and resolve this stuff

Reply:
The space between blocks is not required by the formatter, it's just our convention as documented in Notion.  If you know an easy way to build the rule into our CI, then please help suggest or better implement.